pcdsdevices/state.py

```diff
@@ -362,6 +362,10 @@ class PVStatePositioner(StatePositioner):
         state. You can set this to 'FIRST' instead to use the first state
         found while traversing the `_state_logic` tree. This means an earlier
         state definition can mask a later state definition.
+
+    _state_logic_set_ref : str or None
+        An optional reference to the component that will be used to set some
+        metadata on the state signal if provided.
```

Comment:
So this is something that will have to be set on any future `PVStatePositioner` devices if we want the metadata to carry forward correctly.  Is there somewhere we can broadcast that information?  I don't see natural place in the docs for it yet.  

I also can't imagine there's a good way to dynamically guess at the component name without the creator's help.  Throwing an error in its absence might be a bit heavy handed, since it's only a gui-related issue.

Reply:
I've actually intentionally omitted it in the `MovableStand` class since the state there isn't settable https://github.com/pcdshub/pcdsdevices/blob/613893f5f709d4ba9066a5054004aa10361d1a4b/pcdsdevices/movablestand.py#L32

One thing I was considering was switching this to a required attribute, defaults to `None`, needs to become a `list`. So then we know that you didn't include it if it's still at None, we know that you meant "this is never settable" if it's an empty list, and if there are multiple set PVs we can summarize their write state together. Maybe it's worth doing.

Reply:
And of course, then we could raise an exception during init if it's left at `None`